srv/salt/ceph/configuration/check/default.sls

```diff
@@ -6,7 +6,7 @@ nop:
 {% set client = config + "." + grains['host'] %}
 check {{ config }}:
   file.exists:
-    - name: /srv/salt/ceph/configuration/files/ceph.conf.{{ config }}
+    - name: /srv/salt/ceph/configuration/files/ceph.conf.d/{{ config }}.conf
```

Comment:
earlier we used to ship default config files for `rgw` and `rgw-ssl`, wont this fail if either unless someone explicitly copies into conf.d directory?

Reply:
That's the reason why the `health-rgw.sh --cli --ssl` test fails.

Technically we don't support upgrading from milestone to milestone. We should still be free to introduce non-backwards-compatible changes.

The changes we ship with this patch makes the usage of *.conf files more consistent by moving all of them in the _ceph.conf.d/_ dir.